src/styles/global.css

```
@@ -63,3 +63,28 @@
     @apply h-full max-h-max transition-colors duration-500;
   }
 }
+
+section.footnotes {
+  margin-top: 3rem;
+  font-size: 0.9rem;
+  color: #77838c;
```

ダークモード対応と統一的なデザインのために、色はなるべく定義済みの変数を使っていただけるとありがたいです。

```suggestion
  color: theme(colors.muted.foreground);
```